gensim/models/ensemblelda.py

```diff
@@ -0,0 +1,1315 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Authors: Tobias Brigl <github.com/sezanzeb>, Alex Salles <alex.salles@gmail.com>,
+# Alex Loosley <aloosley@alumni.brown.edu>, Data Reply Munich
+#
+
+
+"""Ensemble Latent Dirichlet Allocation (eLDA), a method of training a topic model ensemble.
+
+Extracts reliable topics that are consistently learned across multiple LDA models. eLDA has the added benefit that
```

Comment:
> Extracts reliable topics that are consistently learned across multiple LDA models.

Why would I want that?

(Asking as a reader; I (Radim) know the answer because I read the article and discussed it with you. But as a casual reader, seeing this description for the first time, I doubt I'd know what this sentence means or why should I care. The motivation should be clearer, assume less.)